legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-physical-plan/src/main/java/org/finos/legend/engine/persistence/components/util/SqlUtils.java

```diff
@@ -55,6 +55,7 @@ public static void logSql(Logger logger, SqlLogging sqlLogging, String sqlBefore
                 break;
             case UNMASKED:
                 logger.info(sqlAfterReplacingPlaceholders);
+                System.out.println(sqlAfterReplacingPlaceholders);
```

Comment:
Let's remove the Sysouts.

Reply:
Sure, removed - was used in testing and accidentally committed